src/routes/profile-route/profile-sidebar.jsx

```diff
@@ -42,6 +42,7 @@ export const ProfileSidebar = () => {
     setToken(null);
     setIsLoggedIn(false);
     client.resetStore();
+    setCurrentUser(null);
```

Comment:
Fixes a bug, where when logging out, the currentUser is not reset. Which meant that, after logging out, I could still click on the movie modal, and see if the last logged in user has clicked either of those 2 buttons.